lib/__tests__/test-stack.ts

```diff
@@ -14,14 +17,52 @@ export class TestStack extends Stack {
   constructor(parent: App, id: string, props: StackProps = { }) {
     super(parent, id, props);
 
+    // dummy publishing secrets. their contet doesn't matter since our pipeline only does
+    // dry run publishing.
+    const npmSecret = new secrets.Secret(this, 'NpmSecret', {
+      generateSecretString: {
+        generateStringKey: 'token',
+        secretStringTemplate: JSON.stringify({}),
+      },
+    });
+
+    const nugetSecret = new secrets.Secret(this, 'NugetSecret', {
+      generateSecretString: {
+        generateStringKey: 'NugetApiKey',
+        secretStringTemplate: JSON.stringify({}),
+      },
+    });
+
+    const mavenSecret = new secrets.Secret(this, 'MavenSecret', {
+      generateSecretString: {
+        generateStringKey: 'password',
+        secretStringTemplate: JSON.stringify({ username: 'user' }),
+      },
+    });
+
+    const pypiSecret = new secrets.Secret(this, 'PypISecret', {
+      generateSecretString: {
+        generateStringKey: 'password',
+        secretStringTemplate: JSON.stringify({ username: '__token__' }),
+      },
+    });
+
+    // these need to be real since they are used before the actual publishing action.
+    const tokenSecret = new secrets.Secret(this, 'TokenSecret1', {
+      secretStringBeta1: secrets.SecretStringValueBeta1.fromToken(ssm.StringParameter.valueForStringParameter(this, 'github-token')),
```

Comment:
I tried using a secure SSM parameter, but its still not supported for `SecretsManager`.

See https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/dynamic-references.html#template-parameters-dynamic-patterns-resources

Would like some input here if you consider this a blocker.